attributes/default.rb

```diff
@@ -61,3 +61,42 @@
 default["confluent"]["schema-registry"]["log4j.properties"]["log4j.appender.ROLLINGFILE.MaxBackupIndex"] = "10"
 default["confluent"]["schema-registry"]["log4j.properties"]["log4j.appender.ROLLINGFILE.layout"] = "org.apache.log4j.PatternLayout"
 default["confluent"]["schema-registry"]["log4j.properties"]["log4j.appender.ROLLINGFILE.layout.ConversionPattern"] = "[%d] %p %m (%c:%L)%n"
+
+default["confluent"]["kafka-connect"]["jar_urls"] = []
+# Expects a hash where they keys are the file name and the values are hashes representing the property_name:value pairs
+default["confluent"]["kafka-connect"]["properties_files"] = {}
+
+default["confluent"]["kafka-connect"]["distributed_mode"] = true
+default["confluent"]["kafka-connect"]["worker_properties_file_name"] = "worker.properties"
```

Comment:
I don't think so. If you are concerned about that could make it more specific like `kafka-connect-worker.properties`